examples/podsecuritypolicy/rbac/bindings.yaml

```diff
@@ -31,7 +31,8 @@ roleRef:
    kind: ClusterRole
    name: restricted-psp-user
 ---
-# edit grants edit role to system:authenticated.
+# edit grants edit role to the groups
+# restricted and privileged.
```

Comment:
I think the whole file is using `restricted` and `privileged`, it is actually saying the group attribute here but not the group name